forge/routes/api/userNotifications.js

```diff
@@ -27,35 +29,46 @@ module.exports = async function (app) {
             }
         }
     }, async (request, reply) => {
-        const paginationOptions = app.getPaginationOptions(request)
-        const notifications = await app.db.models.Notification.forUser(request.session.User, paginationOptions)
-        notifications.notifications = app.db.views.Notification.notificationList(notifications.notifications)
+        const notifications = await getNotifications(app, request)
+
         reply.send(notifications)
     })
 
     // Bulk update
-    // app.put('/', {
-    //     schema: {
-    //         summary: 'Mark notifications as read',
-    //         tags: ['User'],
-    //         body: {
-    //             type: 'object',
-    //             properties: {
-    //                 id: { type: 'string' },
-    //                 read: { type: 'boolean' }
-    //             }
-    //         },
-    //         response: {
-    //             200: {
-    //                 $ref: 'APIStatus'
-    //             },
-    //             '4xx': {
-    //                 $ref: 'APIError'
-    //             }
-    //         }
-    //     }
-    // }, async (request, reply) => {
-    // })
+    app.put('/', {
+        schema: {
+            summary: 'Mark notifications as read',
```

Comment:
I realise this was an existing summary but this feels like it should be less specific

at minimum, it should say "as read or unread" since that functionality is already defined.

personally, i'd go with something like "Update notifications" or "Bulk update notifications" or similar

Reply:
updated the summary entry with the recommended syntax. The route def was uncommented and did not remember to check it again